queue/retry.go

```diff
@@ -244,21 +244,23 @@ func (rh *BasicRetryHandler) waitForJob(ctx context.Context) {
 
 			if err := rh.handleJob(ctx, j); err != nil && ctx.Err() == nil {
 				grip.Error(message.WrapError(err, message.Fields{
-					"message":  "could not retry job",
-					"queue_id": rh.queue.ID(),
-					"job_id":   j.ID(),
-					"service":  "amboy.queue.retry",
+					"message":     "could not retry job",
+					"queue_id":    rh.queue.ID(),
+					"job_id":      j.ID(),
+					"job_attempt": j.RetryInfo().CurrentAttempt,
```

Comment:
Should we add job type here as well?

Reply:
I imagine the specific type of job isn't that important here, since the retry handler logic is all the same regardless of job type.